src/encord_active/lib/metrics/acquisition_functions.py

```diff
@@ -0,0 +1,233 @@
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Optional, Union
+
+import numpy as np
+from PIL import Image
+
+from encord_active.lib.common.iterator import Iterator
+from encord_active.lib.labels.classification import ClassificationType
+from encord_active.lib.labels.object import ObjectShape
+from encord_active.lib.metrics.metric import (
+    AnnotationType,
+    DataType,
+    EmbeddingType,
+    Metric,
+    MetricType,
+)
+from encord_active.lib.metrics.writer import CSVMetricWriter
+
+
+class ModelWrapper:
+    def __init__(self, model):
+        self._model = model
+
+    def prepare_data(self, data_path: Path) -> Optional[Any]:
+        """
+        Reads and prepares a data sample from local storage to feed the model with it.
+
+        Supported data types:
+            * image
+
+        Args:
+            data_path (Path): Path to the data sample.
+
+        Returns:
+            Data sample prepared to be used as input of `self.predict_probabilities()` method.
+        """
+        return [np.asarray(Image.open(data_path)).flatten()]
+
+    def predict_probabilities(self, data) -> Optional[np.ndarray]:
```

Comment:
Do we need that function, can `predict_probabilities` handle that transform for simpler interface?